src/redemptions/approve-list.ts

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import {
+  DataChecksLabel,
+  ExtractionLabel,
+  ImageChecksLabel,
+  JumioTransactionRetrieveResponse,
+  LivenessLabel,
+  SimilarityLabel,
+  UsabilityLabel,
+  WatchlistScreeningLabels,
+} from '../jumio-api/interfaces/jumio-transaction-retrieve';
+
+// ADD APPROVED LABEL SETS HERE AND THEY WILL BE TREATED AS SUCCESS
+// IF A TYPE LABEL IS NOT INCLUDED, IT IS ASSUMED THE VALUES MUST BE 'OK'
+export const BENIGN_FAILURES: ApprovedLabelSet[] = [
+  {
+    maxRiskScore: 50,
+    usabilityLabels: ['NOT_UPLOADED', 'OK'],
```

Comment:
I don't think we should allow this one through, but bad quality is ok.

Reply:
If the livened fails, it is very different than bad quality. The risk score is low on this and Jumio reps suggested if risk was low it is acceptable to pass.